src/util/iterative_restoration.jl

```diff
@@ -42,22 +46,22 @@ function run_iterative_restoration(network, model_constructor, optimizer; repair
     set_component_inactive!(network_mld, get_damaged_items(network_mld))
     _PMs.propagate_topology_status!(network_mld)
 
-    solution_mld = run_mld_strg(network_mld, model_constructor, optimizer, kwargs...)
-    clean_status!(solution_mld["solution"])
-    _PMs.update_data!(network_mld, solution_mld["solution"])
+    result_mld = run_mld_strg(network_mld, model_constructor, optimizer, kwargs...)
+    clean_status!(result_mld["solution"])
+    _PMs.update_data!(network_mld, result_mld["solution"])
 
     ## Turn network into multinetwork solution to merge with solution_iterative
-    mn_network_mld = _PMs.replicate(network_mld,1)
+    mn_network_mld = _PMs.replicate(result_mld["solution"], 1)
```

Comment:
@noahrhodes most of the changes in this PR are just changing the name `solution_*` to `result_*`, but this change from `network_mld` to `result_mld["solution"]` is substantial.  I think this is what is correct here, but please confirm.

Reply:
This should be fine... and then line 51 can be removed
```    
_PMs.update_data!(network_mld, result_mld["solution"])
```

This section only needs to capture a solution for time-period 0, so just this `result_mld["solution"] ` network.

Do the test results change? The iterative tests should cover power/active devices in time period 0.

Reply:
Ok thanks for taking a look.  Tests in PowerModels v0.14 are fine, they are breaking in PowerModels v0.15, most likely due to the change in the convention that inactive components are not reported in the solution anymore.

Reply:
so instead of checking that the status is 0, the solution could check for the component missing from the solution?

Reply:
In the case of calling one `run_*` method, things like `update_data!` work as expected becouse they skip all of the components where the status was `inactive` to start with.   For multi-step algorithms that change status of components and call functions like `propagate_damage_status!`, `set_component_inactive!`, the component status changes need to be accumulated into a shared solution object to so that the result reflects setting status to 0 on components that were removed from the system.